lib/commons.rb

```diff
@@ -84,6 +92,14 @@ def self.build_usage_query(commons_uploads)
     usage_query
   end
 
+  def self.build_info_query(commons_uploads)
+    file_ids = commons_uploads.map(&:id)
+    info_query = { pageids: file_ids,
```

Comment:
I find this pattern a little easier to make sense of. It connects the output of the method with the context in which it gets used. For example, in `self.get_usages`, the first line is `usage_query = build_usage_query commons_uploads`

It makes it easier to quickly see 'oh, yeah, something called "usage_query" is exactly what comes out of `build_usage_query`.

It's not a big deal either way, but to me, being a little more explicit by naming the return value is worth the extra line of code.

(More immediately, the answer for this commit is: "because that's the pattern established in the rest of this class.")